timeless/companies/models.py

```diff
@@ -25,5 +25,12 @@ class Company(TimestampsMixin, DB.Model):
     items = DB.relationship("Item", order_by="Item.id",
                             back_populates="company")
 
+    @validate_required("name", "code")
+
+    def __init__(self, **kwargs):
```

Comment:
@marceloamadeu No need to override attributes with mocked data, just do:
```python
@validate_required("name", "code")
def __init__(self, *args, **kwargs):
    super(Company, self).__init__(**kwargs)
```

By my opinion it looks a bit weird, but this is how `validate_required` implemented.
Please fix it in all places in PR.